.serena/project.yml

```
@@ -0,0 +1,84 @@
+# list of languages for which language servers are started; choose from:
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)